app/frontend/wallet/cardano-wallet.ts

```diff
@@ -232,12 +202,12 @@ const CardanoWallet = (options) => {
     return blockchainExplorer.getTxHistory(addresses)
   }
 
-  async function getAccountInfo() {
+  function getAccountInfo() {
```

Comment:
this seems weird - I would put a comment here explaining why we return undefined (possibly a "TODO" comment so it's not forgotten) or rewrite it

Reply:
We definitely dont need the `getAccountInfo` function, that was just a forgotten piece of code, but the others I left there are there because there are called from the `action.ts` file. Of course only in case of `shelley` implementation of the wallet, but typescript shows an error if the `byron` wallet doesn't have them implemented.

Reply:
then I would make them throw an Error with a message stating that Byron wallet does not support this operation to make sure anybody calling it by mistake realizes it soon enough

Reply:
Yes you are right, I thought about this too, but the typescript had a problem with the function returning void instead of some object. Anyway, I fixed it so it throws Error on this unsupported operations.